examples/multipath.rs

```diff
@@ -0,0 +1,18 @@
+use async_std::task;
+use http_types::Result;
```

Comment:
this should be `tide::Result`. Examples (and tide apps) should not need to directly depend on http_types